src/api/Nhs.Appointments.Api/Functions/RemoveUserFunction.cs

```diff
@@ -22,12 +20,13 @@ public class RemoveUserFunction(
     ILogger<RemoveUserFunction> logger)
     : BaseApiFunction<RemoveUserRequest, RemoveUserResponse>(validator, userContextProvider, logger)
 {
-    [OpenApiOperation(operationId: "RemoveUser", tags: new [] {"Users"}, Summary = "Remove a User")]
-    [OpenApiRequestBody("text/json", typeof(RemoveUserRequest))]
-    [OpenApiSecurity("Api Key", SecuritySchemeType.ApiKey, Name = "Authorization", In = OpenApiSecurityLocationType.Header)]
-    [OpenApiResponseWithBody(statusCode:HttpStatusCode.OK, "text/json", typeof(RemoveUserRequest), Description = "User successfully removed")]
-    [OpenApiResponseWithBody(statusCode:HttpStatusCode.BadRequest, contentType: "text/json", typeof(IEnumerable<ErrorMessageResponseItem>),  Description = "The body of the request is invalid" )]
-    [OpenApiResponseWithBody(statusCode:HttpStatusCode.NotFound, "text/plain", typeof(string), Description = "User did not exist to be removed")]
+    [OpenApiOperation(operationId: "RemoveUser", tags: ["User"], Summary = "Remove a User")]
```

Comment:
I would describe this as "Remove all assigned roles from a user at the specified site"